### PR TITLE
Install current MinGW version for appveyor

### PR DESCRIPTION
**Pull request by mikee47**
Use pre-built release stored in separate github repository.

### CODE REVIEW

File: .appveyor/install.cmd [hunk 1]
```diff
@@ -1,5 +1,9 @@
 REM Windows install script
 
+rmdir /s /q c:\MinGW
+curl -LO https://github.com/mikee47/SmingTools/releases/download/1.0/MinGW.7z
```
Comment (slaff) [inline .appveyor/install.cmd]: @mikee47 Can you add the MinGW toolchain to https://github.com/attachix/sming-toolchains? Hopefully one day we can move all toolchains to SmingHub.

Comment (mikee47) [reply .appveyor/install.cmd]: OK, I've imported the repo to Attachix, rebased and changed the link.

Comment (mikee47) [reply .appveyor/install.cmd]: Looks like it didn't get the release properly. I'll recreate it.

Comment (mikee47) [reply .appveyor/install.cmd]: OK, should be fixed now.

**Comment by slaff** [comment]
@mikee47 Finally we have better control over SmingHub. You should be now the administrator of https://github.com/SmingHub/SmingTools. Please, check if that is the case and then, whenever you have time, change the appveyor script to fetch the current MinGW from SmingHub/SmingTools

**Pull request merged**